test/parallel/test-runner-watch-mode-complex.mjs

```diff
@@ -76,15 +76,17 @@ describe('test runner watch mode with more complex setup', () => {
     runs.push(currentRun);
     currentRun = '';
     const fileToDeletePathLocal = tmpdir.resolve('test-to-delete.mjs');
-    unlinkSync(fileToDeletePathLocal);
+    await new Promise((resolve) => setTimeout(() => {
+      unlinkSync(fileToDeletePathLocal);
+      resolve();
+    }, common.platformTimeout(1000)));
```

Comment:
Can/Should this be simplified with `node:timers/promises`?

Reply:
I didn't think about it, thanks for the suggestion, I'll take a look 😁

Reply:
Would using `node:fs/promises` be enough here? It seems weird to have to put a sync operation in an async one

Reply:
Hey @aduh95, yes, no problem at all.  
The only important logic is the "sleep" after the delete to ensure that the watcher has received the event and completed the test run before proceeding.
Thanks for your feedback, I'll fixit ASAP 😁

Reply:
just update 😁